### DISCUSSION TITLE
[ニュートラル/Neutral] モニカ/Monika(システムに干渉して勝利する)

### DISCUSSION THREAD

**Discussion by yukieiji** [yukieiji/ExtremeRoles#292] [Ideas]
- アイディアが完全に固まりきってないので意見求めも含めてディスカッション化
  - これらの能力、仕様は変更されることがあります

## 能力詳細
- 名前：モニカ
- 色：DokiDoki文芸部のモニカの髪の色
- 陣営：ニュートラル

- 役職コンセプト：AmongUsのシステムに強力に干渉し勝利を目指す役職

- ボタン能力：「抹消」
  - 範囲内のプレイヤー1人を対象にして発動する
  - 対象になった人は会議中の投票券、発言権を奪う => モニカが排除されるとこれらは全て元通りになる
    - 会議中は現世と霊界の間に新しいレイヤー(ゴミ箱)が追加され以下の仕様になる
      - テキストチャット
        - ゴミ箱は現世とゴミ箱が見える
        - 現世はゴミ箱と霊界のチャットは見えない
        - 霊界は全てのチャットが見える
  - タスク中の能力やキルはこれまで通り使える
    - フォトグラファーやボディーガード等のシステムテキストはこれまで通り発言が流れる
    - 通報も出来るシェリフで黙らせるとかも出来なくはない
 - 裏ボタン能力：「ファイアウォール」
   - 一定時間無敵のシールドを張る
   - その間にキルボタンによるキルが発生した場合そのキルを無効化、キルしてきた人を強制暗転 or ノイズエフェクト付与
 - 会議回数は無限回
   - 自身の会議回数が0回の場合、誰かの会議回数を使いその人のボタンとして押させる
   - 全員が0回の場合、自分が押すアニメーション
- サボタージュは直せない 

- 他の陣営への影響
  - 基本的に全陣営の敵になるが、脅威度はこれまでのキル持ちと違い少ない => いわゆる「モニカが落ちた」がわかる + プレイヤーが複数人蘇る
  - この役職の立ち位置的に半キルボタン持ちという微妙な扱いになるため泳がせつつ必要に応じて落とすみたいな戦略も出来るとは思われる

## 未検討事項
- 現状だと恐らくアーソやアンブレくらいの強さだとは思われるので、元ネタ見たく好きな人がいるみたいな感じにして裏の能力を考えてもいい気がする
  - 少人数向けの第三陣営としてはちょうどいいくらいの強さかもしれないが

**Comment by flog4624** [comment]
好きな人がいる設定は元ネタ的に必要だと思います。
元ネタを考えるとモニカを引いたプレイヤー（画面の向こうの人）が好きな人になるのでしょう。
もし他のプレイヤー（こっちは画面内のキャラクター）が好きな人に指定されるのならヤンデレと被ってしまいますから…。
それを踏まえた勝利条件は「自分以外全ての生きているプレイヤーを抹消する」だと考えられます。

裏の能力は、特定のゲームオプションをこのゲーム中だけ強制的に書き換えるというものが思いつきます。
例えば投票先が誰でも見えるようになったり、タスク進捗バーの更新タイミングを変えたりします。
モニカが一定時間静止することで独自のオプション画面を開いて改変するイメージです。

**Reply by yukieiji** [reply]
> 好きな人がいる設定は元ネタ的に必要だと思います。
元ネタを考えるとモニカを引いたプレイヤー（画面の向こうの人）が好きな人になるのでしょう。
もし他のプレイヤー（こっちは画面内のキャラクター）が好きな人に指定されるのならヤンデレと被ってしまいますから…。
それを踏まえた勝利条件は「自分以外全ての生きているプレイヤーを抹消する」だと考えられます。

フレーバーだけを追加するだけなら元ネタに応じず別名で実装すべきと思ってます(フレーバーテキスト追加すれば追加するほどユーザーのテキストが読む量が増えますから)

> 裏の能力は、特定のゲームオプションをこのゲーム中だけ強制的に書き換えるというものが思いつきます。
例えば投票先が誰でも見えるようになったり、タスク進捗バーの更新タイミングを変えたりします。
モニカが一定時間静止することで独自のオプション画面を開いて改変するイメージです。

うーん、これを追加するとモニカのせいでゲーム壊れて使わないってなりそう・・・・

**Comment by ankomon** [comment]
現状はEXR版アーソニストと考えればよろしいでしょうか?
・生存者全員を抹消したら勝利
・抹消をされた対象は会議での発言権や投票権は奪われるが、ゲッサー等の役職能力は行使できる
・抹消された対象は生存扱い(肉体を持ち、ネームプレートが光っている)

**Reply by yukieiji** [reply]
> 現状はEXR版アーソニストと考えればよろしいでしょうか?

会議とかの投票権がない時点でアーソニストとはちょっと違う感じですね

> ・抹消をされた対象は会議での発言権や投票権は奪われるが、ゲッサー等の役職能力は行使できる

ゲッサー等が会議ボタン能力を指すのであれば恐らくできなくするとは思われます
それ以外の部分に関しては出来るとは思ってます

> ・抹消された対象は生存扱い(肉体を持ち、ネームプレートが光っている)

会議中は✕がついている状態でネームプレートが光っている状態を想定してます

**Reply by ankomon** [reply]
好きな人(仮)について、思い浮かんだもの
- 好きな人(仮)
・モニカとのコンビネーション役職のクルー
・(死亡していてもモニカが勝利した場合は一緒に勝利する)
　+モニカが誰か分かる(モニカとは必ず共に勝利する場合なら、モニカを売ってしまうと自らの勝つ方法を1つ潰すことになる為)
・自身をキル能力を使って殺害した相手を下記の恋敵(仮)にする(吊られた場合は、付与されないorモニカ以外の生存者からランダム)
- それに伴うモニカの案
・好きな人に長時間一緒に居た相手を恋敵(仮)(ヤンデレの邪魔者)とする(相当長い時間一緒に居ないと登録されない)
・恋敵に抹消を使用すると、一定時間後に死亡する(ミッショナリーの宣教)
・抹消状態の人数分の票を抹消した相手に投票できる
というのが思い浮かびましたが、どうでしょうか
でも、これ殆どヤンデレですかね…

**Reply by yukieiji** [reply]
> 好きな人(仮)
・モニカとのコンビネーション役職のクルー
・(死亡していてもモニカが勝利した場合は一緒に勝利する)
　+モニカが誰か分かる(モニカとは必ず共に勝利する場合なら、モニカを売ってしまうと自らの勝つ方法を1つ潰すことになる為)
・自身をキル能力を使って殺害した相手を下記の恋敵(仮)にする(吊られた場合は、付与されないorモニカ以外の生存者からランダム)
それに伴うモニカの案
・好きな人に長時間一緒に居た相手を恋敵(仮)(ヤンデレの邪魔者)とする(相当長い時間一緒に居ないと登録されない)
・恋敵に抹消を使用すると、一定時間後に死亡する(ミッショナリーの宣教)
・抹消状態の人数分の票を抹消した相手に投票できる
というのが思い浮かびましたが、どうでしょうか
でも、これ殆どヤンデレですかね…

ほとんどヤンデレですかね
あとはモニカの役職コンセプトである「AmongUsのシステムに強力に干渉し勝利を目指す役職」から大きく離れてる気がしますね

**Reply by ankomon** [reply]
承知しました
